src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -96,42 +97,11 @@ public PurchaseOrder UpdateOrder(PurchaseOrder current, PurchaseOrder updated, I
             }
 
             this.UpdateOrderProperties(current, updated);
-            this.UpdateDetails(current.Details, updated.Details);
-            //Update pl_order_postings? Or just on create? todo investigate
 
+            // Update pl_order_postings? Or just on create? todo investigate
             return current;
         }
 
-        private void UpdateDetails(ICollection<PurchaseOrderDetail> currentDetails, ICollection<PurchaseOrderDetail> updatedDetails)
-        {
-            foreach (var detail in updatedDetails)
-            {
-                var currentDetail = currentDetails.FirstOrDefault(x => x.Line == detail.Line);
-                if (currentDetail != null)
-                {
-                    this.UpdateDetailProperties(currentDetail, detail);
```

Comment:
I'll still need most of this method to update the purchase order details from the po utility, just not foreach with deliveries and the updateDeliveryProperties method

Reply:
(It wasn't referenced even in the interface yet but I will need to use it eventually, or a completed/tested version of it anyway)

Reply:
I think I've accidentally removed this - only meant to remove the delivery part